src/components/node/node.helper.js

```diff
@@ -59,6 +59,59 @@ function buildSvgSymbol(size = CONST.DEFAULT_NODE_SIZE, symbolTypeDesc = CONST.S
         .type(() => _convertTypeToD3Symbol(symbolTypeDesc))();
 }
 
+/**
+ * return dx, dy, and potentially alignmentBaseline and textAnchor props to put label in correct position relative to node
+ * @param {number | undefined} dx - default computed offset of label to the right of the node
+ * @param {'left' | 'right' | 'top' | 'bottom' | 'center' | undefined} labelPosition - user specified position of label relative to node
+ * @returns {{dx: string, dy: string} | {dx: string, dy: string, textAnchor: string, dominantBaseline: string}}
+ * props to put text svg for label in correct spot. default case returns just dx and dy, without textAnchor and dominantBaseline
+ */
+function _getLabelPlacementProps(dx, labelPosition) {
```

Comment:
Please rename to `getLabelPlacementProps`, I use `_` prefix for "private" functions within modules (basically the functions that are not exported).

Reply:
No problem. I don't have access to my dev environment until monday, so I'll put an update in then first thing in the morning. Thanks for the feedback